src/__stories__/cover-hero-story.tsx

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
+import {CoverHero, ButtonPrimary, ButtonLink, Tag, Placeholder, skinVars, Slideshow} from '..';
+import usingVrImg from './images/using-vr.jpg';
+import beachImg from './images/beach.jpg';
+import beachVideo from './videos/beach.mp4';
+
+import type {TagType} from '..';
+
+export default {
+    title: 'Components/Hero/CoverHero',
+    parameters: {
+        fullScreen: true,
+    },
+};
+
+type Args = {
+    background: 'image' | 'video' | 'custom color' | 'color from skin';
+    backgroundColorCustom: string;
+    backgroundColorFromSkin: string;
+    variant: 'default' | 'inverse' | 'alternative';
+    headlineType: TagType;
+    headline: string;
+    pretitle: string;
+    title: string;
+    description: string;
+    withExtra: boolean;
+    withSideExtra: boolean;
+    actions: 'button' | 'link' | 'button and link';
```

Comment:
Can you include the following options?

- None
- Button Primary
- Button Secondary
- Link
- Button Primary and Button Secondary
- Button Primary and Link
- Button Secondary and Link
- Button Primary, Button Secondary and Link

We should have the same config for actions in any stories for buttonGroups

Reply:
None is not allowed by the spec

Reply:
It would be better if we change the spec to allow none actions, we have seen cases where could be useful to allow omit the hero actions:

https://latch.tu.com/usuarios

Does this change have a significant impact?

Reply:
ok